src/index.ts

```diff
@@ -368,6 +371,20 @@ async function challengeResponse(r: NginxHTTPRequest): Promise<void> {
   }
 }
 
+/*
+ * Handler for the `js_periodic` directive that requests /acme/auto periodically
+ * to validate the stored certitificates.
+ */
+async function periodicAuto(): Promise<void> {
+  // TODO should not need this in the final njs-0.8.1 release
+  if (ngx.worker_id !== 0) {
+    return
+  }
+
+  // make the /acme/auto request to localhost
+  await ngx.fetch('http://localhost/acme/auto')
```

Comment:
it would not work if not a standard HTTP port for the server.

Reply:
linking with  https://github.com/nginx/njs-acme/pull/35/ as it similarly sends periodically an HTTP call via layered multiserver aproach

Reply:
What do you think of requiring a URL env var to make this more adaptable? We could keep the URL above as the default.

Reply:
Oh interesting! Happy to implement both and we can compare. I do like the "bespoke server" option for its cleanliness and better security footprint. It would look like this:

```
  server {
    listen 127.0.0.1:10999;
    location = /acme/auto {
      js_content acme.clientAutoMode;
    }
  }
```

Then we could hit localhost:10999 pretty reliably in the periodic, but also provide an option for people to override the URL in case it collides with their environment.

@ivanitskiy @ryepup thoughts?

Reply:
> override the URL in case it collides with their environment.

How does that happen in the "bespoke server" option? If the port is already in use?

Reply:
Oh, they can use whatever port, interface/address, path they want in the "bespoke server" block, and just make sure to refer to it correctly in the NJS_ACME_CLIENT_AUTO_MODE_URL env var.

Reply:
Ok, I gotcha, I was getting confused between what we could do inside njs vs what the user had to do in .conf files. That looks good to me!

Reply:
PR updated with bespoke server and ability to override the request URL.